unitex/src/fr/umlv/unitex/leximir/helper/StatisticOutput.java

```diff
@@ -145,7 +142,7 @@ private void jButtonSetActionPerformed(java.awt.event.ActionEvent evt) {
         if (result == JFileChooser.APPROVE_OPTION) {
 	        File selectedFile = fileChooser.getSelectedFile();
 	        EditorDelas.setStatisticDirectory(selectedFile.getAbsolutePath()+File.separator);
-	        jTextField1.setText(EditorDelas.getStatisticDirectory()+"statisticsTmp.csv");
+	        jTextField1.setText(EditorDelas.getStatisticDirectory()+DictionaryPath.statisticsTmpPath);
```

Comment:
All binary operators except `.` should be separated from their operands by spaces